addons/sale_purchase/tests/test_sale_purchase.py

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from odoo import Command
+from odoo.addons.sale_purchase.tests.common import TestCommonSalePurchaseNoChart
```

Comment:
Useless diff as the imports from odoo addons should be indeed placed last (like it was before) :wink: